python/mxnet/test_utils.py

```diff
@@ -463,9 +463,9 @@ def same(a, b):
     """
     return np.array_equal(a, b)
 
-
 def almost_equal(a, b, rtol=None, atol=None, equal_nan=False):
     """Test if two numpy arrays are almost equal."""
+    # pylint: disable=unexpected-keyword-arg
     return np.allclose(a, b, rtol=get_rtol(rtol), atol=get_atol(atol), equal_nan=equal_nan)
```

Comment:
Please also add `# pylint: enable=unexpected-keyword-arg` so that the rest of the fucntions are checked

Reply:
Done